rollup.config.js

```diff
@@ -1,6 +1,7 @@
 const resolve = require('@rollup/plugin-node-resolve').default;
 const terser = require('rollup-plugin-terser').terser;
 const {author, name, version, homepage, main, module: _module, license} = require('./package.json');
+const copy = require('rollup-plugin-copy');
```

Comment:
I'd rather have the typings file listed in `files` of the package.json and just skip copying it